default.nix

```diff
@@ -9,11 +9,16 @@ in naersk.buildPackage {
     url = "https://github.com/fedimint/minimint";
     ref = "master";
   };
-  gitAllRefs = true;
-  gitSubmodules = true;
+  copyTarget = true;
   buildInputs = [
       pkgs.openssl
       pkgs.pkg-config
       pkgs.perl
   ];
+  shellHook =
+  ''
+    echo "Hello shell"
```

Comment:
Is this necessary?

Reply:
Kept it so it will be easy to track when debugging github actions, but yeah we can remove it